Applications/Spire/Source/Ui/ComboBox.cpp

```diff
@@ -126,6 +126,7 @@ bool ComboBox::eventFilter(QObject* watched, QEvent* event) {
         } else {
           revert_to(m_submission_text, false);
         }
+        event->ignore();
```

Comment:
Why is this event always ignored? It should only be ignored if the ```ComboBox``` takes no action on the key press.

Reply:
Yes, if the ComboBox takes some actions, the event should not be ignored.